frontend/javascripts/oxalis/view/scalebar.tsx

```diff
@@ -80,7 +80,7 @@ function Scalebar({ zoomValue, dataset, viewportWidthInPixels, viewportHeightInP
             ? 16
             : `calc(${scaleBarWidthFactor * 100}% - ${Math.round(
                 ((2 * OUTER_CSS_BORDER) / constants.VIEWPORT_WIDTH) * 100,
-              )}% + ${2 * padding}px)`,
```

Comment:
I think, it was a coincidence that this worked before, because padding is 2px (exactly as the old OUTER_CSS_BORDER value).